scrip/1_somescript.sh

```
@@ -0,0 +1,22 @@
+x=$((100+200))
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)